packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -648,9 +659,9 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     action,
     () => ({
       updateIndicator: updateIndicatorState,
-      updateScrollButtons: updateScrollButtonState,
+      updateScrollButtons: () => {},
```

Comment:
What I meant [here](https://github.com/mui/material-ui/pull/36071#discussion_r1266924350) is that we should retain the method implementation. Having an empty function is equivalent to removing it, which would be a breaking change.